lib/http-invocation.js

```diff
@@ -163,15 +163,15 @@ HttpInvocation.prototype._processArg = function(req, verb, query, accept) {
 
 HttpInvocation.prototype.createRequest = function() {
   var method = this.method;
-  var verb = method.getHttpMethod();
+  var verb = method.getEndpoints()[0].verb;
```

Comment:
perhaps store the endpoint, so that we don't have to repeat `method.getEndpoints[0]`?

```js
var endpoint = method.getEndpoints()[0]);
var verb = endpoints.verb;
//...
req.url = this.base  + endpoint.fullPath;
```

Thoughts?